### DISCUSSION TITLE
Release Management Concepts

### DISCUSSION THREAD

**Discussion by abutalibsh** [jreleaser/jreleaser#354] [General]
Hi,

I admire this project as it helps people standardize their releases. But I was wondering how do I learn the concepts underlying the release management? It's a silly question, but I really have no experience.

Thanks,

Ahmad Abutalib

**Comment by aalmiray** [comment]
The wikipedia page for [Release Management](https://en.wikipedia.org/wiki/Release_management) is a good place to start as it contains links to many concepts.

In particular for JReleaser, the tool is designed to complement existing build tools by providing a model that can be used to generate outputs based on templates and other inputs. JReleaser intersects with the following concepts:

- [Application Release Automation](https://en.wikipedia.org/wiki/Application-release_automation)
- [Release Notes](https://en.wikipedia.org/wiki/Release_notes)
- [Build Automation](https://en.wikipedia.org/wiki/Build_automation)
- [DevOps](https://en.wikipedia.org/wiki/DevOps)

**Reply by abutalibsh** [reply]
I see it's a rabbit hole. But how about you provide an opinionated approach to JReleaser? I mean, I learned a lot from using Spring this way because I can start with minimal setup. After I learned enough, I started customizing.
